project-base/app/MultiDomainEntityClassProvider.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Shopsys;
```

Comment:
I think this class should not be in `app` folder...

Reply:
So where it should be? It does not relate to any model or form or twig..